--- owncloud/apps/files_encryption/lib/crypt.php.wiget	2012-08-30 13:16:03.882982612 +0200
+++ owncloud/apps/files_encryption/lib/crypt.php	2012-08-30 13:16:14.842982378 +0200
@@ -31,7 +31,7 @@
 //  - IMPORTANT! Check if the block lenght of the encrypted data stays the same
 
 
-require_once 'Crypt_Blowfish/Blowfish.php';
+require_once 'Crypt/Blowfish.php';
 
 /**
  * This class is for crypting and decrypting
