--- owncloud/lib/setup.php~	2012-12-19 23:24:24.000000000 +0200
+++ owncloud/lib/setup.php	2013-01-23 00:10:58.370941116 +0200
@@ -6,10 +6,6 @@
 $hasOracle = is_callable('oci_connect');
 $datadir = OC_Config::getValue('datadirectory', OC::$SERVERROOT.'/data');
 
-// Test if  .htaccess is working
-$content = "deny from all";
-file_put_contents(OC::$SERVERROOT.'/data/.htaccess', $content);
-
 $opts = array(
 	'hasSQLite' => $hasSQLite,
 	'hasMySQL' => $hasMySQL,
